static/css/components/page-banner.less

```diff
@@ -56,3 +56,8 @@
     text-align: center;
   }
 }
+@media screen and (max-width: 906px){
```

Comment:
Rather than do this I'd recommend using the mobile first approach as it addresses the largest number of browsers possible (even those that do not support media queries)

.page-banner should be hidden by default and you can use @width-breakpoint-tablet inside static/css/less/breakpoints.less for the media query.